src/en/users.md

```diff
@@ -0,0 +1,100 @@
+Title: Juju and users
+TODO: User abilities, especially owners and regular users
+
+
+# Juju users
+
+Juju has the capacity to manage multiple users. This section covers this
+overall feature.
+
+
+## Introduction
+
+There are 3 kinds of Juju users: controller administrators, model owners, and
+regular users. Each user, no matter the kind, is associated with a controller.
+In particular, the namespace for users are controller-centric; names can be
+duplicated across controllers.
+
+Juju users are not related in any way to the localhost system users; they are
+purely Juju constructs.
+
+A *controller administrator* is a user who has access to the given controller's
```

Comment:
there is no "administrator" in the traditional sense. it's another poor term. each controller has their own and i'm trying to reinforce that notion